code/modules/integrated_electronics/core/saved_circuits.dm

```diff
@@ -151,6 +155,10 @@
 	// Load modified name, if any.
 	if(assembly_params["name"])
 		name = assembly_params["name"]
+		
+	// Load modified description, if any.
+	if(assembly_params["desc"])
+		desc = assembly_params["desc"]
```

Comment:
You also need to check against incorrect values to avoid injection.

Reply:
How might I go about this? I'm pretty sure that the only methods of modifying the description use stripped_input

Reply:
Do the same thing thats done with the name, but for the description. You can use reject_bad_text() instead of reject_bad_name()
https://github.com/r-esistor/tgstation/blob/686e09b3cae2a5a90130107049f25484f993e424/code/modules/integrated_electronics/core/saved_circuits.dm#L147